src/extension.ts

```diff
@@ -0,0 +1,151 @@
+import * as os from 'os';
+import { workspace, ExtensionContext, window, commands } from 'vscode';
+
+import {
+  LanguageClient,
+  LanguageClientOptions,
+  ServerOptions,
+  TransportKind
+} from 'vscode-languageclient/node';
+import { promisify } from 'util';
+import { execFile } from 'child_process';
+
+let client: LanguageClient;
+
+const outputChannel = window.createOutputChannel('Wasp Language Server');
+
+export async function activate(context: ExtensionContext) {
+  console.log('..Extension "vscode-wasp" is now active..');
+
+  const config = workspace.getConfiguration('wasp');
```

Comment:
Where is this string `'wasp'` coming from?

Reply:
From `package.json`, in `contributes.configuration`. It's based on the property names. I agree that it's kind of weird, but this is the way vscode makes you do it.

Reply:
Maybe mention it in the comment.